packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -74,6 +74,12 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       activeStateEnabled,
       focusStateEnabled,
       hoverStateEnabled,
+      onInput: ({ component }): void => {
```

Comment:
Let's get `event` argument with correct type of Event. Kindly fix it

Reply:
I have removed some code, it's necessary right now.

[332595f30abb7fcb0abcc63677fca5a99e430962]